azurerm/internal/services/sentinel/sentinel_data_connector_aws_cloud_trail.go

```diff
@@ -96,7 +96,7 @@ func resourceSentinelDataConnectorAwsCloudTrailCreateUpdate(d *schema.ResourceDa
 				},
 			},
 		},
-		Kind: securityinsight.KindAmazonWebServicesCloudTrail,
+		Kind: securityinsight.KindBasicDataConnectorKindAmazonWebServicesCloudTrail,
```

Comment:
this is a breaking change in a minor version of the SDK?

Reply:
@ArcturusZhang Would you please clarify?

Reply:
We currently do not block the breaking changes in preview packages in minor versions